docs/source/en/using-diffusers/stable_diffusion_jax_how_to.mdx

```diff
@@ -202,50 +213,3 @@ image_grid(images, 2, 4)
 ```
 
 ![img](https://huggingface.co/datasets/YiYiXu/test-doc-assets/resolve/main/stable_diffusion_jax_how_to_cell_43_output_0.jpeg)
-
-
-## How does parallelization work?
```

Comment:
This was very valuable info no?

Reply:
I think it'd be nice to put this section in the Conceptual Guides (you can call it JAX parallelization or something like that), and add a link to it from this doc. To me, this info is more explanatory and good for deepening a user's understanding but not as helpful when they're just looking for the steps on how to do something.